src/loginServices.js

```diff
@@ -110,15 +115,50 @@ export function getBindings(okapiUrl, store, tenant) {
     });
 }
 
+function getErrorMessage(error = {}) {
+  const { code = 'default.error', parameters } = error;
```

Comment:
What is the value of the parameters when they are empty? I am wondering whether it can also be assigned to the default value like:

```
parameters = {}
```